src/StatusQ/Core/Theme/Theme.qml

```diff
@@ -4,6 +4,9 @@ import QtQuick 2.13
 
 QtObject {
     id: appTheme
+    // Replace it with:
+    //      property QtObject palette: StatusLightTheme {}
+    // for reloading
```

Comment:
Is this the "potential" fix for hot reloading theme palettes?

Let's put this in a separate commit then, ideally together with the changes that introduce theme switching (otherwise there's nothing to be fixed in the first place)